chain/tendermint/src/runtime/abi.rs

```diff
@@ -0,0 +1,722 @@
+#[path = "../protobuf/fig.tendermint.codec.v1.rs"]
+mod pbcodec;
+
+use anyhow::anyhow;
+use graph::runtime::{
+    asc_new, AscHeap, AscIndexId, AscPtr, AscType, DeterministicHostError, ToAscObj,
+};
+use graph_runtime_wasm::asc_abi::class::{Array, Uint8Array};
+
+use crate::codec;
+
+pub(crate) use super::generated::*;
+
+impl ToAscObj<AscEventList> for codec::EventList {
```

Comment:
I feel bad that you had to write all of this 😓 

This should be automatically generated. I'll fix this soon 🙏